FacialRecog.py

```diff
@@ -148,6 +171,64 @@ def compare_encodings(frame_encodings, all_encodings):
 """
 returns a list of all cameras in db
 """
+
+
 def all_cameras():
     cameras = get_all_cameras("not_api_call")
     return cameras
+
+
+"""
+given a camera from db, returns a frame
+from that camera
+"""
+
+
+def get_frame_from_camera(camera):
+    feed = cv2.VideoCapture("http://" + str(camera['ip']) + "/video.mjpg")
+    ret, img = feed.read()
+    frame = cv2.resize(img, (0, 0), fx=0.75, fy=0.75)
+    return frame
+
+
+"""
+given a frame and a list of encodings,
+returns a list of comparisons between faces
+in the frame and the list of encodings
+"""
+
+
+def compare_frame_and_encodings(frame, encodings):
+    frame_encodings = get_face_encodings(frame)
+    comparison_results = compare_encodings(frame_encodings, encodings)
+    return comparison_results
+
+
+"""
+given a list of comparisons, list of ids, camera, frame,
+processes comparisons
+"""
+
+
+def process_comparisons(comparisons, ids, camera, frame):
+    if comparisons is not None:
+        print(comparisons)
```

Comment:
We might want to delete the print statements
```suggestion
```

Reply:
just left them in there so if yall ran it you would see what i meant, but ya we can remove them